Session/Conversations/Message Cells/Content Views/LinkPreviewView.swift

```diff
@@ -18,6 +18,8 @@ final class LinkPreviewView: UIView {
     private lazy var imageViewContainerHeightConstraint = imageView.set(.height, to: 100)
 
     // MARK: UI Components
+    
+    public var previewView: UIView { hStackView }
```

Comment:
Doesn't seem to be used anywhere. Do you want to remove it or keep it for refactor in the future?

Reply:
This one is used in `ConversationVC+Interaction` `L:1052`:
```
let linkPreviewPoint: CGPoint = visibleCell.convert(cellLocation, to: visibleCell.linkPreviewView?.previewView)
```

The `LinkPreviewView` actually contains the `TappableLabel` so in order to determine whether the touch was made within the "top" section or the body text I need to be able to reference the `hStackView` (and thought that `previewView` was a bit easier to understand from another file)

I think the goal would be to refactor the `VisibleMessageCell` entirely to use SwiftUI at which point we can probably standardise the different layouts and look to properly recycle UI elements instead of recreating them every time (which will hopefully give a bit of a performance boost)